pdns/dnsdist-console.cc

```diff
@@ -446,6 +446,7 @@ const std::vector<ConsoleKeyword> g_consoleKeywords{
   { "setUDPMultipleMessagesVectorSize", true, "n", "set the size of the vector passed to recvmmsg() to receive UDP messages. Default to 1 which means that the feature is disabled and recvmsg() is used instead" },
   { "setUDPTimeout", true, "n", "set the maximum time dnsdist will wait for a response from a backend over UDP, in seconds" },
   { "setVerboseHealthChecks", true, "bool", "set whether health check errors will be logged" },
+  { "setWebserverConfig", true, "password [, apiKey [, customHeaders ]]", "Updates webserver configuration" },
```

Comment:
Perhaps this function should take a table? A user could then update one of the items without updating the others. e.g.

```lua
setWebserverConfig({password: "hello"})
```
```lua
setWebserverConfig({apiKey: "hello2"})
```

Reply:
this means using `setWebserverConfig({apiKey: ""})` to disable apiKey authentication. Is that something we want ? Otherwise I agree.